### PR TITLE
setup.py: dependencies is now a directory

### PR DESCRIPTION
**Pull request by nirbheek**
Fixes setup.py install. Reported by @ebassi on IRC.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -93,4 +94,4 @@ def run(self):
       long_description='''Meson is a cross-platform build system designed to be both as
 fast and as user friendly as possible. It supports many languages and compilers, including
 GCC, Clang and Visual Studio. Its build definitions are written in a simple non-turing
-complete DSL.''')
+omplete DSL.''')
```
Comment (ebassi) [inline setup.py]: Not sure this is intentional.

Comment (nirbheek) [inline setup.py]: HAH. Oops.

**Comment by QuLogic** [comment]
Oops. Maybe we should add a test that the installed copy works? But really, why don't we use something like `find_packages` and not worry about this sort of thing?

**Comment by nirbheek** [comment]
We could run the tests in some configurations/platforms with an installed Meson, and/or do the find_packages thing,

**Pull request merged**